### PR TITLE
Add rectangle draw feature to Sketch on Map sample

### PR DESCRIPTION
**Pull request by duffh**
# Description

Updated Xamarin.Forms, WPF and WinUI Sketch on Map samples. 
 - Reworked UI on WPF and WinUI
 - Updated BasemapStyle and Viewpoint

### CODE REVIEW

File: src/WinUI/ArcGISRuntime.WinUI.Viewer/Samples/GraphicsOverlay/SketchOnMap/SketchOnMap.xaml.cs [hunk 1]
```diff
@@ -53,60 +52,66 @@ private void Initialize()
...
             // Set the sketch editor as the page's data context
             DataContext = MyMapView.SketchEditor;
         }
 
         #region Graphic and symbol helpers
-
-        private Graphic CreateGraphic(Esri.ArcGISRuntime.Geometry.Geometry geometry)
+        private Graphic SaveGraphic(Geometry geometry)
         {
+            //Geometry geometry = _sketchOverlay.SelectedGraphics.FirstOrDefault().Geometry;
```
Comment (nCastle1) [inline src/WinUI/ArcGISRuntime.WinUI.Viewer/Samples/GraphicsOverlay/SketchOnMap/SketchOnMap.xaml.cs:65]: Dead code should be removed

**Review by nCastle1 — COMMENTED** [review]
I just noticed this PR is draft after I got halfway through reviewing it.

Please confirm that the approach to button icons works well for WinUI in both light and dark mode.

File: src/WPF/ArcGISRuntime.WPF.Viewer/Samples/GraphicsOverlay/SketchOnMap/SketchOnMap.xaml.cs [hunk 2]
```diff
@@ -133,16 +138,27 @@ private async Task<Graphic> GetGraphicAsync()
...
+            string mode = (sender as Button).CommandParameter.ToString();
+
+            // Check if the command parameter is defined in the SketchCreationMode enumerator
+            if (Enum.IsDefined(typeof(SketchCreationMode), mode))
+            {
+                _ = DrawButtonClick((SketchCreationMode)Enum.Parse(typeof(SketchCreationMode), mode));
+            }
+        }
+
+        private async Task DrawButtonClick(SketchCreationMode creationMode)
```
Comment (ZackAllen) [inline src/WPF/ArcGISRuntime.WPF.Viewer/Samples/GraphicsOverlay/SketchOnMap/SketchOnMap.xaml.cs:153]: I recommend renaming this method.
```suggestion
        private async Task StartSketch(SketchCreationMode creationMode)
```

File: src/WPF/ArcGISRuntime.WPF.Viewer/Samples/GraphicsOverlay/SketchOnMap/SketchOnMap.xaml.cs [hunk 3]
```diff
@@ -43,8 +44,8 @@ public SketchOnMap()
 
         private void Initialize()
         {
-            // Create a light gray canvas map
-            Map myMap = new Map(BasemapStyle.ArcGISLightGray);
+            // Create a map
```
Comment (ZackAllen) [inline src/WPF/ArcGISRuntime.WPF.Viewer/Samples/GraphicsOverlay/SketchOnMap/SketchOnMap.xaml.cs:47]: Comments should end with a period.

**Review by ZackAllen — CHANGES_REQUESTED** [review]
I recommend adding this sample to the featured samples list.
Also run codemaid on every .cs file in here.

File: src/Forms/Shared/Samples/GraphicsOverlay/SketchOnMap/SketchOnMap.xaml.cs [hunk 4]
```diff
@@ -117,76 +114,77 @@ private Graphic CreateGraphic(Esri.ArcGISRuntime.Geometry.Geometry geometry)
                     }
             }
 
-            // pass back a new graphic with the appropriate symbol
+            // pass back a new graphic with the appropriate symbol.
```
Comment (asantana917) [inline src/Forms/Shared/Samples/GraphicsOverlay/SketchOnMap/SketchOnMap.xaml.cs:117]: This should be capitalize.

**Review by ZackAllen — APPROVED** [review]

**Review by asantana917 — APPROVED** [review]

**Pull request merged**